db/migrate/20190612195503_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[5.2]
+  def change
+    create_table :posts do |t|
+      t.string :caption
+      t.integer :status
```

Comment:
have you considered putting a default value on `:status`?

Reply:
Yes, but that we plan to do when we implement `Create`method